workflow/tests/ACCA_Examples/Long_Residence.xml

```diff
@@ -167,8 +167,8 @@
           </Window>
           <Window>
             <SystemIdentifier id='Window2'/>
-            <Area>8.0</Area>
```

Comment:
Window areas didn't sum up with the same values as manual J example, I inspected each window surfaces and reconfigured a bit here. Not sure if it will break the unit test or reduce the discrepancy. Let's see the ci return.